spec/spec_helper.rb

```diff
@@ -73,7 +74,21 @@
   #   EvmSpecHelper.log_ruby_object_usage
   # end
 
-  # Preconfigure and auto-tag specs in the automation subdirectory a la rspec-rails
+  config.include VMDBConfigurationHelper
+
+  config.define_derived_metadata(:file_path => /spec\/lib\/miq_automation_engine\/models/) do |metadata|
+    metadata[:type] ||= :model
+  end
```

Comment:
Most of the diff in this file is from my organizing the very-hard-to-read groups of helper inclusions by metadata. The above block and the `require` at the top are the only two additions to this file.